docker/Dockerfile-build-backend.jvm

```diff
@@ -1,42 +1,33 @@
-####
```

Comment:
Why remove the comments? They can be useful for someone who would be unfamiliar with that kind of file.

Reply:
They were auto generated and the same in every quarkus project. Also the dockerfile is 99% the same as in every Quarkus project generated via https://code.quarkus.io/

I would say they are not useful here. I mean, when "could be useful for someone", we should also comment our java code very often as this code is less intuitive than this dockerfile :)

Reply:
They are generated for a good reason: because they're considered helpful for people unfamiliar with Dockerfiles. I still think we should keep them, not for us because we're all familiar with these files, but for someone who would join the team in the future.

> we should also comment our java code very often

Yes we should and it is actually already done in many classes.

Reply:
added again

Reply:
ps: I personally prefer to write the cleanest possible software so its self explaining and we don't need coments :-D

Reply:
> ps: I personally prefer to write the cleanest possible software so its self explaining and we don't need coments :-D

I completely agree but it's not always possible unfortunately 😃